.github/workflows/UpdateSiteCertificate.yml

```diff
@@ -0,0 +1,99 @@
+name: Update Site Certificate
+
+on:
+  schedule:
+    - cron: "1 1 1 * *"
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  update-site-certificate:
+    runs-on: ubuntu-latest
+    environment: WebCertificate
+
+    steps:
+      - name: Azure login
+        uses: azure/login@v1
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+
+      - uses: actions/checkout@v4
+        with:
+          repository: "EssentialCSharp/EssentialCSharp.Web.CertbotDnsGoDaddy"
+          token: ${{ secrets.CERTBOTDNSGODADDY_PAT }}
+          # If testing, change this to the "staging" branch to not use up your Let's Encrypt quota for the week (which is very very small)
+          ref: "main"
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.12'
+          cache: 'pip'
+
+      - name: Retrieve the key and add it to a file
+        env:
+          APIKEY_BASE64: ${{ secrets.GODADDY_APIKEY_BASE64 }}
+        run: |
+          echo $APIKEY_BASE64 | base64 --decode > godaddycreds.ini
+
+      - name: Run Certbot 
+        run: |
+          pip install --upgrade pip
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -r requirements.txt
+          chmod 600 godaddycreds.ini
+          sudo .venv/bin/python -m main
+          deactivate
+
+      - name: Clear API File
+        if: '!cancelled()'
+        run: |
+          rm *ini
+
+      - name: Create Private Certificate
+        env:
+          CERT_PASSWORD: ${{ secrets.CERT_PASSWORD }}
+        run: |
+          sudo chmod 0755 /etc/letsencrypt/{live,archive}
+          sudo chown $USER: /etc/letsencrypt/live/essentialcsharp.com/privkey.pem
+          sudo chmod 0640 /etc/letsencrypt/live/essentialcsharp.com/privkey.pem
+          mkdir ./tmp/sandbox -p
+          cd ./tmp/sandbox/
```

Comment:
```suggestion
```

Reply:
I'll go adjust the other paths as well for this